tools/check_format.py

```diff
@@ -94,7 +113,13 @@ def fixFileContents(file_path):
   for line in fileinput.input(file_path, inplace=True):
     # Strip double space after '.'  This may prove overenthusiastic and need to
     # be restricted to comments and metadata files but works for now.
-    sys.stdout.write("%s" % (line.replace('.  ', '. ')))
+    line = line.replace('.  ', '. ')
+
+    # Fix incorrect protobuf namespace references.
+    for error, replacement in PROTOBUF_TYPE_ERRORS.items():
+      line = line.replace(error, replacement)
```

Comment:
the PROTOBUF_TYPE_ERRORS should also be iterated in the 'check' flow, not just the 'fix' flow  This will get easier once #2921 is merged, as that introduces a line-by-line check algorithm analogous to the 'fix' algorithm.  @akonradi can you add that once #2921 is merged?

Reply:
Sure, will do.

Reply:
+1 thanks, was just about to comment on this.

Reply:
Opened #2979 to add to the 'check' flow.